README.md

```diff
@@ -32,14 +32,23 @@ There are two ways to use `memory_profiler`:
 ### Command Line
 
 The easiest way to use memory_profiler is via the command line, which requires no modifications to your program. The basic usage is:
+
+```
+$ ruby-memory-profiler [options] run [--] command [command-options]
 ```
-$ ruby-memory-profiler [options] <script.rb> [--] [script-options]
```

Comment:
CLI allows to profile single files or whole command. Since single files can be profiled as commands via simply `... ruby script.rb ...`, I removed that ability from the readme to make it less confusing and for people to use the unified style.